news/2019-05-01-bn-and-qat-restructure.md

```diff
@@ -0,0 +1,36 @@
+
+The primary focus of probationary BNs will be to nominate beatmaps, and so will not have the ability to disqualify beatmaps or take part in mediating vetoes. Moreover, they will be unable to nominate beatmaps with more than one game mode, and a beatmap cannot be qualified by two probationary BNs. New members of the BNs will be placed in probation for approximately one month before their performance is reviewed by the NAT. They will then be promoted to a regular BN, remain in probation, or be removed from the group completely. Members of the regular BNs may also be demoted to probation if their work is found to not meet standards.
+
+The [Beatmap Nominator rules](https://osu.ppy.sh/help/wiki/People/Beatmap_Nominators/Rules) have been updated accordingly for their new tasks. Moreover, work on a score system as a guideline to evaluate Beatmap Nominators starts now and you can expect a new proposal detailing our plans for that soon!
+
+Additionally to this, BN applications will be open indefinitely! We have a [new website](http://bn.mappersguild.com/) to deal with BN affairs and mapping related reports. Here, you will be able to submit BN applications, as well as reports against beatmaps and members of the BN and NAT. BN applications are expected to open on the new site tomorrow!
+
+BN applications will still work roughly the same as last time, where you are required to pass an activity check and to complete a test on the [Ranking Criteria](https://osu.ppy.sh/help/wiki/Ranking_Criteria) to complete your application. Additionally, there will be a 90 day cooldown for submitting applications once you submit. The results of your application should be sent out as soon as they are processed, however, we expect a larger influx of applications during the initial weeks. Check out [the website](http://bn.mappersguild.com/) to submit an application to join the BN!
+
+As osu! is a community driven game, these changes were made to give members of the community more control over what reaches the ranked section. We expect to see improvements in the way beatmap quality is managed, particularly with controversial maps, where the outcome will be more in line with the desires of the community.
+
+With these changes, we unfortunately have had to say farewell to [Fycho](https://osu.ppy.sh/users/1876867), [Gabe](https://osu.ppy.sh/users/654108), [Hobbes2](https://osu.ppy.sh/users/8157492), [Kuron-kun](https://osu.ppy.sh/users/2697284), [Lanturn](https://osu.ppy.sh/users/1446665), [p3n](https://osu.ppy.sh/users/123703), [Regraz](https://osu.ppy.sh/users/3076909), [Raiden](https://osu.ppy.sh/users/2239480), [JBHyperion](https://osu.ppy.sh/users/4879508), [Protastic101](https://osu.ppy.sh/users/6712747), [-Kazu-](https://osu.ppy.sh/users/920861), and [Noffy](https://osu.ppy.sh/users/1541323), who will not be joining the NAT. Nonetheless, we greatly appreciate their time in the QAT and thank them all for their hard work up until now.
```

Comment:
feel like the "we unfortunately have had to say farewell" bit sounds a little pretentious because you're pretending that it wasn't your own decision. it also makes it sound like this is a negative change. make it direct instead
```suggestion
With these changes, the following members of the QAT will not be joining the NAT: [Fycho](https://osu.ppy.sh/users/1876867), [Gabe](https://osu.ppy.sh/users/654108), [Hobbes2](https://osu.ppy.sh/users/8157492), [Kuron-kun](https://osu.ppy.sh/users/2697284), [Lanturn](https://osu.ppy.sh/users/1446665), [p3n](https://osu.ppy.sh/users/123703), [Regraz](https://osu.ppy.sh/users/3076909), [Raiden](https://osu.ppy.sh/users/2239480), [JBHyperion](https://osu.ppy.sh/users/4879508), [Protastic101](https://osu.ppy.sh/users/6712747), [-Kazu-](https://osu.ppy.sh/users/920861), and [Noffy](https://osu.ppy.sh/users/1541323). We greatly appreciate their time in the QAT and thank them all for their hard work up until now. Most of them will still be active as BNs or GMT.
```

Reply:
This kinda makes it sound impersonal, which I feel also has that sort of "pretentious" connotation.

Reply:
is it supposed to be personal? (maybe I'm not understanding what you mean by that)

Reply:
What I don't want is for us to come off as heartless robots, essentially.

Reply:
rereading my suggestion a few times I'm still not getting that impression, but in any case the current line should probably change so can you write your own alternative if mine is no good

Reply:
Tried thinking of an alternative way to write things and yeah, yours is pretty much the best way to go about it.
```suggestion
With these changes, the following members of the QAT will not be joining the NAT: [Fycho](https://osu.ppy.sh/users/1876867), [Gabe](https://osu.ppy.sh/users/654108), [Hobbes2](https://osu.ppy.sh/users/8157492), [Kuron-kun](https://osu.ppy.sh/users/2697284), [Lanturn](https://osu.ppy.sh/users/1446665), [p3n](https://osu.ppy.sh/users/123703), [Regraz](https://osu.ppy.sh/users/3076909), [Raiden](https://osu.ppy.sh/users/2239480), [JBHyperion](https://osu.ppy.sh/users/4879508), [Protastic101](https://osu.ppy.sh/users/6712747), [-Kazu-](https://osu.ppy.sh/users/920861), and [Noffy](https://osu.ppy.sh/users/1541323). Nonetheless, we greatly appreciate their time in the QAT and thank them all for their hard work up until now. Most of them will still be active as BNs or GMT.
```